repondeur/setup.py

```diff
@@ -8,6 +8,7 @@
     "SQLAlchemy",
     "transaction",
     "zope.sqlalchemy",
+    "Jinja2>=2.9",
```

Comment:
Isn't it already installed via `pyramid-jinja2`? Or is that for a particular new feature? (if yes document?)